source/hassio/haos_common_tasks.markdown

```diff
@@ -0,0 +1,133 @@
+
+## Configuring access to files
+
+One of the first things to take care of after installing Home Assistant OS is to provide yourself access to files. There are several add-ons commonly used for this, and most users employ a mix of various add-ons. Default directories on the host are mapped to the add-ons so that they can be accessed by the services any particular add-on might provide. On the host system these directories exist on the `/data` partition at `/mnt/data/supervisor/`. 
+
+Using any of the add-ons listed below,the following directories are made available for access:
+
+- addons
+- backup
+- config
+- media
+- share
+- ssl
+
+---
+
+### Installing and using the Samba add-on
+
+The Samba add-on creates smb shares which can be accessed from another computer. You can also edit files using the editor of your preference from your client computer.
+
+Configuring the add-on is straight-forward. You only need to set a user and password of your choice on the configuration page, save, and then start the add-on. The add-on will not start without setting a password!
+
+For connecting from Windows 10, you can enter the IP address or hostname of your Home Assistant instance in File Explorer's address bar with two backslashes, as shown in the example screenshot.
+
+![explorer address bar](TBD image of file explorer address bar.jpg)
+
+You should then be prompted for the credentials you entered in the Samba add-on configuration. You also have the option of having the credentials stored so that you do not need to enter them again. After that, you'll have access to the directories which you can then mount as a drive or pin to Quick Access.
+
+![directories](TBD image of directories.jpg)
+
+For OS X, connecting to the shares is a matter of using the Finder menu > Go > Connect to Server...
+You would then enter the IP address of your Home Assistant OS instance as `smb://your.ha.ip.address` and enter your credentials when prompted.
+
+---
+
+### Installing and using the SSH add-on (requires enabling advanced mode for the HA user)
+
+The Terminal & SSH add-on provides access over an ssh connection, and also inludes nano and vi editors.  In addition, it provides access to the Home Assistant Command Line Interface (CLI) which provides custom commands for checking logs, stopping and starting Home Assistant and add-ons, creating/restoring snapshots, and more. (See [Home Assistant via Command Line](https://www.home-assistant.io/hassio/commandline/) for further info). The Terminal & SSH add-on does *not* provide access to the underlying host file system.
+
+To use the add-on, enter a password or public key on the configuration page, then save and start the add-on. In order to access from an ssh client, a port needs to be entered in the network section of the add-on's configuration page.
```

Comment:
Mention that we need to install it and how.

Reply:
Or at the top we should mention that all of these add-ons can be found in the add-on store. In that case, you need to this, and similar paragraphs with a mention that it needs to be installed:
```suggestion
To use the add-on after installing it from the add-on store, enter a password or public key on the configuration page, then save and start the add-on. In order to access from an ssh client, a port needs to be entered in the network section of the add-on's configuration page.
```

Reply:
I did a little of both. It is mentioned within "Configuring access to files" at the top, and individually for each add-on.